ceno_zkvm/src/scheme.rs

```diff
@@ -156,7 +156,8 @@ pub struct ZKVMProof<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> {
     pub raw_pi: Vec<Vec<E::BaseField>>,
     // the evaluation of raw_pi.
     pub pi_evals: Vec<E>,
-    pub chip_proofs: BTreeMap<usize, ZKVMChipProof<E>>,
+    // each circuit may have multiple proof instances
+    pub chip_proofs: Vec<(usize, ZKVMChipProof<E>)>,
```

Comment:
soundness: i think we need to have some mechanism to check, w.g. maximal proofs allow for a chip. E.g. what if prover fill same proof multiple times

Reply:
I made this part to be `BTreeMap<usize, Vec<ZKVMChipProof<E>>>` then at least constrain OMC init proof can only be `ONE` proof

Reply:
> what if prover fill same proof multiple times

Then the offline memory checking `prod_r == prod_w` must fail.